js/build.js

```diff
@@ -1,23 +1,23 @@
 var container = $('[data-onboarding-id]');
-var id = $(container).data('onboarding-id');
+var id = $('.fl-widget-instance').data('id');
```

Comment:
@squallstar When we first time added an onboarding component application couldn't find id(thrown an error in console) for it and when we added slider or image to it they are disappeared from the screen. That why I added id for onboarding component via $('.fl-widget-instance').data('id'). The sliderId is needed for correct event handling newly added slider, the ability to slide images because the slider is added and onboarding component should not have one id.